HACKING.md

```diff
@@ -45,15 +45,6 @@ The following should work in a fresh Git clone:
       sudo dnf builddep --spec $TEMPFILE && \
       rm $TEMPFILE
 
-In addition, for testing, the following dependencies are required:
```

Comment:
These are all test dependencies. Test dependencies are not included in build dependencies.
Build dependencies are the packages you need to build cockpit from sources not run the tests.